src/components/header.js

```diff
@@ -1,17 +1,15 @@
-import { Link } from "gatsby"
 import PropTypes from "prop-types"
 import React from "react"
+import { HeaderContainer, HeaderLink, HeaderLinkTitle, HeaderLinkImage } from "../theme"
 import logoGif from '../images/unhandled-exception-club-logo-transparent.gif'
 
 const Header = ({ siteTitle }) => (
-  <header>
-    <Link to="/" className="header-link">
-      <img src={logoGif} alt="unhandled exception club logo" />
-      <div>
-        {siteTitle}
-      </div>
-    </Link>
-  </header>
+  <HeaderContainer>
+    <HeaderLink to="/" className="header-link">
```

Comment:
```suggestion
    <HeaderLink to="/">
```

we don't need header-link class anymore